pkg/storage/bloom/v1/bloom_tokenizer.go

```diff
@@ -159,4 +206,20 @@ func (bt *BloomTokenizer) PopulateSeriesWithBloom(seriesWithBloom *SeriesWithBlo
 			Checksum: chunks[idx].Checksum,
 		})
 	} // for each chunk
+
+	endTime := time.Now().UnixMilli()
+
+	fillRatio := seriesWithBloom.Bloom.ScalableBloomFilter.FillRatio()
+	bt.metrics.hammingWeightRatio.Observe(fillRatio)
+	bt.metrics.estimatedCount.Observe(
+		float64(estimatedCount(seriesWithBloom.Bloom.ScalableBloomFilter.Capacity(), fillRatio)),
+	)
+	bt.metrics.bloomSize.Observe(float64(seriesWithBloom.Bloom.ScalableBloomFilter.Capacity() / 8))
```

Comment:
could you please move out this magic number to a variable to explain why we divide the value by 8?